definitions/v2/mircrew.yml

```diff
@@ -40,6 +40,7 @@ caps:
     - {id: 53, cat: Movies, desc: "Richieste Film"}
     - {id: 54, cat: TV, desc: "Richieste Serie TV"}
     - {id: 55, cat: Movies, desc: "Richieste Video diverse"}
+    - {id: 57, cat: Movies, desc: "Richieste Video Soddisfatte"}
```

Comment:
Should this be on v1 also?

Reply:
No. They are not a v1 indexer.

utilizing this indexer in Prowlarr is blocked by the Selctors / V2 PR

Reply:
We didn't backport them? It's a few lines right

Reply:
It'd be needing to rewrite the download implementation To not use selectors.

that applies to any of the only v2 indexers.

Reply:
Yea seems they use the selectors to hack the link on this one, so good. I made v1 for elite, so just that update needed.